sklearn/ensemble/_weight_boosting.py

```diff
@@ -5,14 +5,14 @@
 
 The module structure is the following:
 
-- The ``BaseWeightBoosting`` base class implements a common ``fit`` method
+- The :class:`~sklearn.ensemble.BaseWeightBoosting` base class implements a common ``fit`` method
```

Comment:
This should be reverted, since `BaseWeightBoosting` is not a public class and would not link.